addons/package_hierarchy/models/stock_quant_package.py

```diff
@@ -246,3 +244,37 @@ def get_key(x):
             ):
                 return False
         return True
+
+    def _get_all_package_links(self):
+        self.ensure_one()
+        HierarchyLink = self.env["package.hierarchy.link"]
```

Comment:
Nit: Rename to `PackageHierarchyLink` so it's consistent with the other method in this MR.